tests/devices/i10/test_i10Apple2.py

```diff
@@ -349,10 +349,10 @@ async def test_I10Apple2_pol_set(
     mock_id_pol.id_ref()._energy_set(energy)
     if pol == "dsf":
         with pytest.raises(ValueError):
-            await mock_id_pol.set(pol)
+            await mock_id_pol.set(Pol(pol))
```

Comment:
Could: You can just change the `parametrize` to use `Pol.LH` rather than the `str` "lh"?